apps/frontend/src/components/global/upload_tabs/splunk/SplunkReader.vue

```diff
@@ -49,6 +49,31 @@
             <v-icon pr-2>mdi-github-circle</v-icon>
             Splunk Configuration
           </v-btn>
+          <br />
+          <span>
+            The Splunk platform accepts any type of data. In particular, it
+            works with all IT streaming
```

Comment:
what is IT streaming?

Reply:
It is when you create a pipe to an endpoint that continuously pushes its output.